packages/material-ui-lab/src/TimelineItemTail/TimelineItemTail.js

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { withStyles } from '@material-ui/core/styles';
+
+export const styles = (theme) => ({
+  /* Styles applied to the root element. */
+  root: {
+    width: 2,
+    backgroundColor: theme.palette.grey[300],
+    transform: 'translate(-50%)',
+    height: '100%',
+    position: 'absolute',
+    top: theme.spacing(2),
+    boxSizing: 'border-box',
```

Comment:
Changing the box sizing doesn't seem to change layout. Do we need this CSS property?

```suggestion
```

Reply:
I restructured the styles this is not needed anymore